src/main/java/net/snowflake/ingest/streaming/internal/ParquetRowBuffer.java

```diff
@@ -210,8 +210,8 @@ private float addRow(
       throw new SFException(
```

Comment:
Alternatively, is it worth having a list of longs that we store in a `SFException` that can be accessed via something like `getRowIndexes`? This will reduce the need to parse an error message which is error prone.

Reply:
We already doing that with CONTINUE, this is for ABORT where we thrown an exception directly